src/Honeybee_FileExplorer.py

```diff
@@ -0,0 +1,49 @@
+#Developed by: Sarith Subramniam, Penn State
+#sarith@sarith.in
+#Demo: https://youtu.be/KR5tRepXgRA
+"""
+    
+    Locate a file/directory in windows explorer
+    
+    Args:
+        Destination: File path or Directory path
+        Open: Boolean value. Default is True
+"""
+
+ghenv.Component.Name = "Explorer"
+ghenv.Component.NickName = 'Explorer'
+ghenv.Component.Message = 'Ver Alpha\n27_July_2015'
+ghenv.Component.Category = "PSUAE"
+ghenv.Component.SubCategory = "2 | Utilities"
```

Comment:
Maybe we should start a new subcategory for IES components but we can start by putting them under WIP until we figure it out.